docs/etcd/demo_cluster_setup.md

```diff
@@ -0,0 +1,272 @@
+# Set up a demo etcd cluster
+
+This guide will help you set up an example etcd cluster that you can use with CN-WAN Operator.
+
+In the first part, *Start the cluster*, we will install and start etcd. The second part, *Make it more secure*, is not mandatory but **highly** suggested.
+
+**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production.  
+We strongly encourage you to follow more thorough guides if you want to use etcd in production.
+
+## Start the cluster
+
+This section will guide you through installing, setting up and start a demo etcd cluster.
+
+Please note that while it will create a ready-to-use and working cluster, we **strongly** encourage you to read etcd's [official documentation](https://etcd.io/docs/v3.4.0/) to learn how to make it more robust, resilient and secure in case you want to use it in production.
+
+### Requirements
+
+Please make sure the machine where you plan to install etcd is reachable by the CN-WAN Operator, so that you can later follow the other guides on this documentation.
+
+Also, although this won't be much of problem, make sure that ports `2379` and `2380` are free on that machine, as those are well-known ports [assigned by IANA](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=etcd) to etcd and the ones that will be used throughout these guides. Nonetheless, you can also specify the port on installation phase.
+
+Please note that as of this writing, `v3.4.14` is the latest version of etcd and it has no installation/binaries for *Windows* and therefore we will be using *Linux*/*Mac* shell commands from now on. If you plan to use *Windows* you will need to either use [Docker](https://docs.docker.com/docker-for-windows/install/), [wsl](https://docs.microsoft.com/en-us/windows/wsl/install-win10) or use a virtual machine.
+
+### Get the IP address
+
+First, get the IP address of the machine where you want to etcd server to listen for connections from. Depending on your operating system, this could mean running either `ip a` or `ifconfig -a` or even go through the machine's operating system settings if it has GUI.
+
+The CN-WAN Operator will use this address to connect to etcd, so make sure it is reachable.
+
+Export the following variables, containing the IP address and the name of this cluster:
+
+```bash
+export CLUSTER_IP=<your-IP-address>
+export CLUSTER_NAME=<name>
+```
```

Comment:
I am rebasing those to `THIS_IP` and `THIS_MACHINE` which are the same name etcd uses on their demo guide.

Reply:
Yes, but while that works for installing etcd on its own, it can still be confusing in a setup with multiple components. Having `ETCD` somewhere in the name like `AWS` has it in all its variables is like namespace and avoids confusion completely.